mibitracker/request_helpers.py

```diff
@@ -492,6 +492,25 @@ def upload_channel(self, image_id, image_file, filename=None):
                                      'object that does not have a name')
             return self._upload_channel(image_id, image_file, filename)
 
+    def upload_channel_from_mibiimage(self, image, channel_label, image_id):
+        """Uploads a channel from a MibiImage to the MibiTracker.
+
+        The image passed as argument should contain only the channel that needs
+        to be uploaded.
+
+        Args:
+            image: a MibiImage with the channel to upload.
+            channel_label: the name of the channel to upload.
+            image_id: The integer id of the image to associate the channel with.
+
+        Returns:
+            The response from the MibiTracker after uploading the file.
+        """
+        buf = io.BytesIO()
+        skio.imsave(buf, image)
```

Comment:
If you want to upload a single channel from a MibiImage, I think a modification to the existing `upload_channel` function to allow a wide variety of inputs would be more appropriate.

Also note that `skio.imsave` will not work on a MibiImage itself, but would work on a selected _channel_ of a MibiImage such as `image['dsDNA']`.